guides/common/attributes-satellite.adoc

```diff
@@ -92,6 +92,7 @@
 :ProjectX: Satellite{nbsp}6
 :ProjectXY: Satellite{nbsp}{ProductVersionRepoTitle}
 :provision-script: kickstart
+:RHC: {Team} Cloud
```

Comment:
```suggestion
:RHCloud: {Team} Hybrid Cloud Console
```
I'd prefer to go with this if you agree. I'd definitely use the official product name.